week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,18 @@
+/*HELLO DANIEL
+HELLO IRINA
+HELLO MIMI
+HELLO ROB
+HELLO YOHANNES */
+
+function shoutOuts(name) {
+  return "Hello" + name.toUpperCase();
```

Comment:
It's probably better to put the space after "Hello" - i.e. `"Hello "` rather than having to pass it with every name.